gtfs-realtime/spec/en/reference.md

```diff
@@ -170,8 +171,8 @@ Note that the update can describe a trip that has already completed.To this end,
 
 Timing information for a single predicted event (either arrival or departure). Timing consists of delay and/or estimated time, and uncertainty.
 
-*   delay should be used when the prediction is given relative to some existing schedule in GTFS.
-*   time should be given whether there is a predicted schedule or not. If both time and delay are specified, time will take precedence (although normally, time, if given for a scheduled trip, should be equal to scheduled time in GTFS + delay).
+*   delay should be used when the prediction is given relative to some existing schedule in GTFS. For added or replacement trips, the existence of the delay field indicates that there is a scheduled time, calculated by time - delay, at the stop for the trip, in addition to the estimated time.
```

Comment:
```suggestion
*   delay should be used when the prediction is given relative to some existing schedule in GTFS. For added or replacement trips, the existence of the delay field indicates that there is a scheduled time, calculated by `time - delay`, at the stop for the trip, in addition to the estimated time.
```

Reply:
@gcamp The alternative to this type of context-dependent math is to introduce a new field for that, maybe `scheduledTime`. I'm not sure if like it better or not. Do you have an opinion?

Reply:
Another alternative would be to simply move this change to another PR so this one doesn't get sidetracked into this discussion.

Reply:
This is related to #503 but I think it's better for this to supersede that.

If we add a field called `scheduledTime`, it should behave in the same way as a time in static GTFS for scheduled trips, but I think it will complicate matter by introducing a source of conflict similar to schedule trips where `time` and `delay` don't match, which `time` takes precedence.